lib/watir/locators/element/locator.rb

```diff
@@ -154,7 +154,7 @@ def wd_find_all_by(how, what)
         def fetch_value(element, how)
           case how
           when :text
-            element.text
+            Watir::Element.new(@query_scope, element: element).text_content
```

Comment:
Would it be bad to do:

    Watir::Element.new(@query_scope, element: element).send(:execute_js, :getTextContent, element).strip

It adds a bit of code duplication, but it would save an extra wire call (ie waiting for the element to exist even though we just retrieved the element).

Reply:
I agree this is better. I'll add it.